src/coreclr/jit/fginline.cpp

```diff
@@ -589,22 +589,14 @@ class SubstitutePlaceholdersAndDevirtualizeWalker : public GenTreeVisitor<Substi
                 }
 #endif // DEBUG
 
-                CORINFO_CONTEXT_HANDLE context                = nullptr;
+                assert((call->gtCallMoreFlags & GTF_CALL_M_HAS_LATE_DEVIRT_INFO) != 0);
```

Comment:
What is the point of this flag now?

Reply:
A leftover that should be unnecessary, but I hesitated to remove it. I can remove it if you would like.

Reply:
Yes, this should be removed if it is unnecessary.